library/js/oeUI/universalTooltip.js

```diff
@@ -0,0 +1,17 @@
+$(function () {
```

Comment:
This script needs a license/author/copyright header

Reply:
It was showing up on the page source, Do you want me to enclose it in `<?php ?>` tags

Reply:
No, just use js comment lines

Reply:
here's an example in js:
https://github.com/openemr/openemr/blob/e10a69185a5843ec2c0f1616bc03b16bf6af3aa6/library/js/vendors/validate/validate_extend.js#L1-L19

Reply:
The javadoc syntax is visible in the page source however using the phpdoc syntax i.e. the same thing enclosed in `<?php ?>` tags prevents that from happening

Reply:
I wouldn't worry about the header showing up in the page source. Would rec not using php tags in pure js since some servers may have unpredictable issues.